src/context_store/ingestion/classifier.py

```
@@ -81,7 +81,7 @@
     r"is a type of",
     r"is an abbreviation",
     r"stands for",
-    r"# ",  # Markdown 見出し (ドキュメント)
+    r"# ",  # Markdown 見出し（ドキュメント）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)